apps/src/blockly/addons/cdoUtils.js

```diff
@@ -106,9 +109,168 @@ export function getUserTheme(themeOption) {
 }
 
 export function getCode(workspace) {
+  // Begin collecting data about our ability to correctly serialize a workspace in JSON format.
+  if (
+    experiments.isEnabled(experiments.BLOCKLY_JSON) ||
+    Math.random() <= 0.05
```

Comment:
Is there an experiment config we could use to modify this value without a deploy (ex. `DCDO.blocky_experiment_sample_rate`), just in case we want to be able to modify without a code change? (Is that worth doing?) Otherwise, could we make it a constant?

Reply:
I’ve added a DCDO Boolean flag that can be used to turn the entire feature on or off. I went with that because it seems to be how we usually use DCDO flags and that made me feel it would be most intuitive for any engineer that suddenly needed to go in and change the value.  I could also modify this to be a rate like you suggest if that seems better, so let me know what you think.